lib/msf/core/post/windows/runas.rb

```diff
@@ -19,7 +19,7 @@ def initialize(info = {})
         'Compat' => {
           'Meterpreter' => {
             'Commands' => %w[
-              stdapi_railgun_api*
+              stdapi_railgun_*
```

Comment:
So this changes it to require `stdapi_railgun_memread` and `stdapi_railgun_memwrite` in addition to `stdapi_railgun_api` and `stdapi_railgun_api_multi`. I don't see where the memory related functions are needed here though. There's only two implementations of railgun right now, Windows and Python. This would mark only the Windows one as compatible since the Python one lacks the direct memory related functions.

Same comment regarding the `file_system.rb` changes from `stdapi_raigun_api*` to `stdapi_railgun_*`. The `accounts.rb` mixin on the other hand does need to remain `stdapi_railgun_*` (as it's already defined) because of the use of the `railgun.util` functions `read_data`, `read_string` and `unpack_pointer`.